website/content/contributors/getting-started.md

```diff
@@ -0,0 +1,111 @@
+
+### Read the Documentation
+Before you start contributing to a project, it's important to read the documentation. Most projects have a README file that provides an overview of the project, how to set it up, and how to contribute. Read the documentation carefully to make sure you understand the project's goals, coding standards, and contribution guidelines. 
+
+It's also important to invest time in reading the project's mailing list threads, community forum discussions, and meeting minutes, as well as reviewing open and closed issues and pull requests in the project repository. By doing so, you can gain a deeper understanding of the project's context, goals, and challenges, and develop a sense of the community's culture and values. This knowledge can help you ask more informed and relevant questions when reaching out to the community, and increase your chances of getting a helpful and constructive response.
+
+### A language of their own
+The open source community can sometimes be overwhelming with its acronyms and technical terms. Newcomers might find it difficult to keep up with the jargon. However, there are ways to overcome this, such as taking the time to research and ask questions. Also, the Cloud Native Computing Foundation (CNCF) provides a helpful [glossary of commonly used terms](https://glossary.cncf.io/abstraction/) in the cloud native landscape. By utilizing this resource and being proactive in seeking out information, newcomers can better understand the community and its practices. Open source communities aim to be collaborative and inclusive, and its members are generally welcoming to those who are willing to learn and contribute.
+
+### Start Small
+When you're first getting started with open source, it's important to start small. Look for easy issues to fix or small features to add. This will help you get familiar with the project's codebase and build your confidence as a contributor. As you become more experienced, you can take on more challenging tasks.
+
+### Collaborate with Others
+Open source is all about collaboration. Don't be afraid to reach out to other contributors for help or feedback. You can also collaborate with other contributors by reviewing their code, providing feedback, and testing their changes. This will help you build relationships with other contributors and improve your coding or other skills you bring to the community. Contributing to an open source project doesn't require permission. Once you've done sufficient research and have a clear understanding of where your skills can help the community, don't hesitate to dive in and start contributing.
```

Comment:
```suggestion
Open source is all about collaboration. Don't be afraid to reach out to other contributors for help or feedback. You can also collaborate with other contributors by reviewing their code, providing feedback, and testing their changes. This will help you build relationships with other contributors and improve your coding or other skills you bring to the community. Contributing to an open source project doesn't require permission. Once you've done sufficient research and clearly understand where your skills can help the community, don't hesitate to dive in and start contributing.
```

Reply:
Added. Thanks